process_css_extend_template.xml

```diff
@@ -4,38 +4,34 @@
 --><project xmlns:dita="http://dita-ot.sourceforge.net" name="fox.jason.extend.css">
     
     <target name="extend.css.init">
+        <condition property="user.csspath" value="" else="${args.csspath}/">
+            <or>
+                <not>
+                    <isset property="args.csspath"/>
+                </not>
+                <isabsolute path="${args.csspath}"/>
+            </or>
+        </condition>
         <mkdir dir="${dita.temp.dir}"/>
-        <tempfile deleteonexit="true" destdir="${dita.temp.dir}" property="extend.css.file" suffix=".css"/>
-        <touch file="${extend.css.file}"/>
+        <tempfile deleteonexit="true" destdir="${dita.temp.dir}" property="extend.css.file" suffix=".css" createfile="yes"/>
     </target>
     
     <target name="extend.css.copy" >
         <copy tofile="${output.dir}/${user.csspath}/common-extended.css" file="${extend.css.file}"/>
     </target>
-    
-    <target name="extend.css" depends="extend-css-check-transtype, extend-css-set-properties" if="enable.extend.css">
-        <antcall target="call.extend.css" inheritRefs="true"/>
-    </target>
-    
+
     <target name="extend-css-check-transtype">
         <condition property="enable.extend.css">
```

Comment:
is.html is a pretty generic name. I'd be wary of using such names in an ant environment where all properties from all plugins are everywhere

Reply:
Added `<local>` guard